itests/update_asset_info_api_test.go

```diff
@@ -0,0 +1,317 @@
+package itests
+
+import (
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+	f "github.com/wavesplatform/gowaves/itests/fixtures"
+	"github.com/wavesplatform/gowaves/itests/testdata"
+	utl "github.com/wavesplatform/gowaves/itests/utilities"
+	"github.com/wavesplatform/gowaves/itests/utilities/issue_utilities"
+	"github.com/wavesplatform/gowaves/itests/utilities/update_asset_info_utilities"
+	"github.com/wavesplatform/gowaves/pkg/crypto"
+)
+
+type UpdateAssetInfoTxApiSuite struct {
+	f.BaseSuite
+}
+
+func (suite *UpdateAssetInfoTxApiSuite) TestUpdateAssetInfoTxApiReissuableTokenPositive() {
+	versions := update_asset_info_utilities.GetVersions(&suite.BaseSuite)
+	issue_versions := issue_utilities.GetVersions(&suite.BaseSuite)
+	waitForTx := true
+	for _, v := range versions {
+		for _, iv := range issue_versions {
+			reissuable := testdata.GetCommonIssueData(&suite.BaseSuite).Reissuable
+			itx := issue_utilities.IssueBroadcastWithTestData(&suite.BaseSuite, reissuable, iv, waitForTx)
+			tdmatrix := testdata.GetUpdateAssetInfoPositiveDataMatrix(&suite.BaseSuite, itx.TxID)
+			for name, td := range tdmatrix {
+				height := utl.GetHeight(&suite.BaseSuite)
+				caseName := utl.GetTestcaseNameWithVersion(name, v)
```

Comment:
I think `iv` should be also included in the testcase name.

Reply:
Fix